Description: jxl-oxide creates would require to add at least 8 jxl-* crates
 Disabling the feature for now.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -118,3 +118,3 @@
-[dependencies.jxl-oxide]
-version = "0.12.0"
-features = ["image"]
+# [dependencies.jxl-oxide]
+# version = "0.12.0"
+# features = ["image"]
--- a/src/common/image.rs
+++ b/src/common/image.rs
@@ -18,3 +18,3 @@
-use jxl_oxide::image::BitDepth;
-use jxl_oxide::integration::JxlDecoder;
-use jxl_oxide::{JxlImage, PixelFormat};
+// use jxl_oxide::image::BitDepth;
+// use jxl_oxide::integration::JxlDecoder;
+// use jxl_oxide::{JxlImage, PixelFormat};
@@ -29,1 +29,1 @@
-use crate::common::consts::{HEIC_EXTENSIONS, IMAGE_RS_EXTENSIONS, IMAGE_RS_SIMILAR_IMAGES_EXTENSIONS, JXL_IMAGE_EXTENSIONS, RAW_IMAGE_EXTENSIONS};
+use crate::common::consts::{HEIC_EXTENSIONS, IMAGE_RS_EXTENSIONS, IMAGE_RS_SIMILAR_IMAGES_EXTENSIONS, RAW_IMAGE_EXTENSIONS};
@@ -40,2 +40,3 @@
 
+/*
 pub(crate) fn get_jxl_image(path: &str) -> anyhow::Result<DynamicImage> {
@@ -62,4 +63,5 @@
     Ok(image)
 }
+*/
 
 pub fn get_dynamic_image_from_path(path: &str) -> Result<DynamicImage, String> {
@@ -76,2 +78,2 @@
-        } else if JXL_IMAGE_EXTENSIONS.iter().any(|ext| path_lower.ends_with(ext)) {
-            get_jxl_image(path).map_err(|e| format!("Cannot open jxl image file \"{path}\": {e}"))
+        // } else if JXL_IMAGE_EXTENSIONS.iter().any(|ext| path_lower.ends_with(ext)) {
+            // get_jxl_image(path).map_err(|e| format!("Cannot open jxl image file \"{path}\": {e}"))
@@ -178,10 +180,10 @@
     };
     let extension_str = extension.to_string_lossy().to_lowercase();
     #[cfg(feature = "heif")]
-    let allowed_extensions = &[IMAGE_RS_EXTENSIONS, RAW_IMAGE_EXTENSIONS, JXL_IMAGE_EXTENSIONS, HEIC_EXTENSIONS].concat();
+    let allowed_extensions = &[IMAGE_RS_EXTENSIONS, RAW_IMAGE_EXTENSIONS, HEIC_EXTENSIONS].concat();
 
     #[cfg(not(feature = "heif"))]
-    let allowed_extensions = &[IMAGE_RS_EXTENSIONS, RAW_IMAGE_EXTENSIONS, JXL_IMAGE_EXTENSIONS].concat();
+    let allowed_extensions = &[IMAGE_RS_EXTENSIONS, RAW_IMAGE_EXTENSIONS].concat();
 
     allowed_extensions.iter().any(|ext| extension_str.ends_with(ext))
 }
